### ISSUE TITLE
Processing's tweak mode suggestion

### ISSUE THREAD

**Issue by dartfnm** [imp5imp5/dasbox#16]
Просто интересно, а получится ли на dascript'e сделать подобие TweakMode как в Processing'e
Как на видео: https://youtu.be/b56gTG87gCk?t=248

Если dascript хранит инструкции в виде нодового дерева, то по идее можно вывести в отдельном окне редактор скрипта (через imgui+stb_textedit) и покрутить в нём значения переменных как слайдеры в реальном времени ?!

**Comment by imp5imp5** [comment]
Да, константы можно менять. Плагин дебаггера для VS Code так умеет делать. (только я не знаю, работает ли новый дебаггер с версией daScript, которая сейчас в dasbox)
